fuse-exfat/exfat-utils: new recipes.

OscarL opened the pull request:
Leaving "extfat-utils" disabled, its commmands mostly overlap with the ones from "exfatprogs" (already available on repos), and that seems more actively maintained.

Note: the fuse-exfat module, while working mostly OK, causes a continuable KDL panic due to an assertion failure on unmount.

https://review.haiku-os.org/c/haiku/+/9743 is my attempt to fix that on Haiku's side.

----

Opening as Draft for now, most likely until after beta6 gets released. Still, should help test the mentioned changeset at least.

Also... needs testing on x86_32 bits.

waddlesplash commented:
The assertion failure was new to the nightlies, so beta5 shouldn't have that problem. There have been a bunch of other fixes to userlandfs since beta5 though, so it may not work too well until then anyway.

OscarL committed:
fuse-exfat/exfat-utils: new recipes.

Leaving "extfat-utils" disabled, its commmands mostly overlap with
the ones from "exfatprogs" (already available on repos), and that
seems more actively maintained.

On nightlies `fuse-exfat`, requires hrev59089 or newer (to avoid a
continuable KDL panic due to an assertion failure on unmount).

OscarL commented:
Would be a good idea to add a `fuse:foobar` or `userlandfs:foobar` PROVIDES to recipes with fuse/userland modules? (eg: `fuse:exfat`, `fuse:squashfs`)

pulkomandy commented:
I don't think so:
- that's only useful if something else depends on it, in this case I don't see what would depend on that
- everything in provides get added to the repository description file, so that pkgman can find it. That results in the rather large repository file that pkgman re-downloads everytime its changed. Let's try to not bloat it even more?
